File: BOJ/BOJ - 10989.py
"""
author : Kwak Daeun
github : https://github.com/daeunkwak

title : 수 정렬하기 3
description : 정렬
"""

import sys

N = int(sys.stdin.readline())
num = []
res = [0] * 1001

for i in range (N) :
    num = int(sys.stdin.readline())
    res[num] += 1

for i in range (len(res)) :
        for j in range (res[i]) :
            print(i)

File: BOJ/BOJ - 10989.java
/**
 * author : https://github.com/daeunkwak
 * date : 2022-01-06
 * title : 수 정렬하기 3
 * description : 정렬
 */


import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;


public class BOJ_10989 {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[n];

        for(int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(br.readLine());
        }

        Arrays.sort(arr);

        StringBuilder sb = new StringBuilder();
        for(int i = 0; i < n; i++){
            sb.append(arr[i]).append('\n');
        }

        System.out.println(sb);

    }
}
